disnake/utils.py

```diff
@@ -316,6 +317,32 @@ def oauth_url(
     return url
 
 
+def parse_token(token: str) -> Tuple[int, datetime.datetime, bytes]:
+    """Parse a token into its parts
+    
+    Returns 
+    
+    Parameters
+    -----------
+    token: :class:`token`
```

Comment:
do we have token class?
```suggestion
    token: :class:`str`
```

Reply:
Totes my bad, will fix it once I get the chance